doc/source/train/user-guides/using-gpus.rst

```diff
@@ -8,11 +8,18 @@ be done in a few lines of code.
 The main interface for configuring scale and resources
 is the :class:`~ray.train.ScalingConfig`.
 
-Scaling Configurations in Train (``ScalingConfig``)
----------------------------------------------------
+:class:`~ray.train.ScalingConfig` overview
+------------------------------------------
+Ray Train's :class:`~ray.train.ScalingConfig` configures the number of training workers
+and the resources they should use.
 
-The scaling configuration specifies distributed training properties like the number of workers or the
-resources per worker.
+A *worker* here refers to a *training worker*. Technically, it's a
+:ref:`Ray Actor <actor-key-concept>` that runs your training loop.
+
+The :class:`Trainer <ray.train.trainer.BaseTrainer>` object you instantiate in the
+training script contains the settings to run your training. When you call
+:meth:`Trainer.fit() <ray.train.trainer.BaseTrainer.fit>`, it will be scheduled
+as a :ref:`Ray Actor <actor-key-concept>`. It can then also use resources.
 
```

Comment:
Not able to leave the comment on the exact lines since it was not edited in this PR, but I feel like the code snippet should just be moved to an example in the API reference. 1. I feel like we're missing an example in the API reference. 2. I don't think we should include more advanced confgs like `placement_strategy` in the introduction of this guide.